src/controllers/requestController.ts

```diff
@@ -149,7 +149,7 @@ export class RequestController {
                 if (this._restClientSettings.previewResponseInUntitledDocument) {
                     this._textDocumentView.render(response, previewColumn);
                 } else {
-                    this._webview.render(response, previewColumn);
+                    this._webview.render(response, httpRequest, previewColumn);

```

Comment:
No need to pass the `httpRequest` argument, since the underlying request already set in the response object. The only problem is that, the value of `request` field in response object doesn't set the `requestVariableCacheKey` value, so I suggest to set in the https://github.com/Huachao/vscode-restclient/blob/master/src/utils/httpClient.ts#L106

Reply:
Thanks, will check it out.